Makefile

```diff
@@ -159,3 +160,56 @@ sv: clean_tmp mkdirs copy_assets build_sv finalize
 tr: clean_tmp mkdirs copy_assets build_tr finalize
 zh-CN: clean_tmp mkdirs copy_assets build_zh-CN finalize
 zh-TW: clean_tmp mkdirs copy_assets build_zh-TW finalize
+
+# Localizations
+
+localize_ar:
+	$(LIVESCRIPT_BIN) $(LOCALIZE_SCRIPT) ar
```

Comment:
Thanks! Didn't know Make could do that. Will send a PR simplifying the existing build targets too